crates/wasm/wasm-test-components/src/impls/bootstrap/build_chain_driver.rs

```diff
@@ -136,6 +152,50 @@ where
             .await
             .map_err(Bootstrap::raise_error)?;
 
+        let chain_home_dir = bootstrap.chain_store_dir();
+
+        // Write the wallet secret as a file so that a tester can use it during manual tests
+        let wasm_addresses_file = Runtime::join_file_path(
+            chain_home_dir,
+            &Runtime::file_path_from_string("wasm-addresses.env"),
+        );
+
+        let mut lines = vec![];
+
+        let sender: Signer = chain.get_default_signer().account().into();
+
+        // Upload and instantiate additional Wasm contracts
+        for additional_wasm_code in bootstrap.wasm_additional_byte_codes().iter() {
+            // TODO: Set correct access type
+            let code_id = chain
+                .upload_wasm_contract(
+                    additional_wasm_code.as_slice(),
+                    &sender.as_ref().to_string(),
+                    &WasmAccessConfig::Everybody,
+                )
+                .await
+                .map_err(Bootstrap::raise_error)?;
+            let contract_address = chain
+                .instantiate_wasm_contract(
+                    &sender.as_ref().to_string(),
+                    bootstrap.governance_proposal_authority(),
+                    b"{}".to_vec().as_slice(),
```

Comment:
I think, you can just do,


```suggestion
                    b"{}",
```